File: webstore/webstore-java/src/main/java/com/renan/webstore/util/MessageUtil.java
package com.renan.webstore.util;

public class MessageUtil {
    
    private static final String MSE01 = "MSE01"; // An error ocurred on the operation.
    private static final String MSE02 = "MSE02"; // Error on values properties.
    private static final String MSE03 = "MSE03"; // Invalid Token.
    private static final String MSE04 = "MSE04"; // Invalid User or Password.
    private static final String MSE05 = "MSE05"; // E-mail already exists.
    private static final String MSE06 = "MSE06"; // E-mail not found.
    private static final String MSE07 = "MSE07"; // User not activated.
    private static final String MSE08 = "MSE08"; // User does not have the required privileges.
    
    public static String getErrorMessage() { return MSE01; }
    
    public static String getErrorMessageInputValues() {
        return MSE02;
    }
    
    public static String getErrorMessageToken() {
        return MSE03;
    }
    
    public static String getErrorMessageLogin() {
        return MSE04;
    }
    
    public static String getErrorMessageEmailAlreadyExists() {
        return MSE05;
    }
    
    public static String getErrorMessageUserNotFound() {
        return MSE06;
    }
    
    public static String getErrorMessageUserNotActive() {
        return MSE07;
    }
    
    public static String getErrorMessageDontHavePrivilege() { return MSE08; } // User does not have the required privileges.
}

File: webstore/webstore-kotlin/src/main/kotlin/com/renan/webstore/util/MessageUtil.kt
package com.renan.webstore.util

private const val MSE01 = "MSE01" // An error occurred on the operation.
private const val MSE02 = "MSE02" // Error on values properties.
private const val MSE03 = "MSE03" // Invalid Token.
private const val MSE04 = "MSE04" // Invalid User or Password.
private const val MSE05 = "MSE05" // E-mail already exists.
private const val MSE06 = "MSE06" // E-mail not found.
private const val MSE07 = "MSE07" // User not activated.
private const val MSE08 = "MSE08" // User does not have the required privileges.

fun getErrorMessage() =  MSE01
fun getErrorMessageInputValues() = MSE02
fun getErrorMessageToken() = MSE03
fun getErrorMessageLogin() = MSE04
fun getErrorMessageEmailAlreadyExists() = MSE05
fun getErrorMessageUserNotFound() = MSE06
fun getErrorMessageUserNotActive() = MSE07
fun getErrorMessageDoNotHavePrivilege() = MSE08